analysis/statistics/handling_cost.py

```diff
@@ -0,0 +1,12 @@
+from typing import Tuple
+
+from heuristic.classes import Problem, Solution
+
+
+def handling_cost(solution: Solution) -> Tuple[str, float]:
+    """
+    Handling cost per volume unit.
+    """
+    problem = Problem()
+
+    return "handling_cost", problem.handling_cost
```

Comment:
Or just `return "handling", Problem().handling_cost`

Reply:
It returns a property object if i do this.